bundles/org.openhab.binding.echonetlite/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,378 @@
+				<option value="Veranda/balcony 2">Veranda/balcony 2</option>
+				<option value="Veranda/balcony 3">Veranda/balcony 3</option>
+				<option value="Veranda/balcony 4">Veranda/balcony 4</option>
+				<option value="Veranda/balcony 5">Veranda/balcony 5</option>
+				<option value="Veranda/balcony 6">Veranda/balcony 6</option>
+				<option value="Veranda/balcony 7">Veranda/balcony 7</option>
+
+				<option value="Others">"Others"</option>
+				<option value="Others 1">Others 1</option>
+				<option value="Others 2">Others 2</option>
+				<option value="Others 3">Others 3</option>
+				<option value="Others 4">Others 4</option>
+				<option value="Others 5">Others 5</option>
+				<option value="Others 6">Others 6</option>
+				<option value="Others 7">Others 7</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="standardVersionInformation">
+		<item-type>String</item-type>
+		<label>Standard Version Information</label>
+		<description>Standard Version Information</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="identificationNumber">
+		<item-type>String</item-type>
+		<label>Identification Number</label>
+		<description>Identification Number</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="manufacturerFaultCode">
+		<item-type>String</item-type>
+		<label>Manufacturer Fault Code</label>
+		<description>Manufacturer Fault Code</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="faultStatus">
+		<item-type>Switch</item-type>
+		<label>Fault Status</label>
+		<description>Fault Status</description>
+		<category>Alarm</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="faultDescription">
+		<item-type>String</item-type>
+		<label>Fault Description</label>
+		<description>Fault Description</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="manufacturerCode">
+		<item-type>String</item-type>
+		<label>Manufacturer Code</label>
+		<description>Manufacturer Code</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="businessFacilityCode">
+		<item-type>String</item-type>
+		<label>Business Facility Code</label>
+		<description>Business Facility Code</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="powerSavingOperationSetting">
+		<item-type>Switch</item-type>
+		<label>Power Saving Operation Setting</label>
+		<description>Power Saving Operation Setting</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<channel-type id="cumulativeOperatingTime">
+		<item-type>Number</item-type>
+		<label>Cumulative Operating Time</label>
+		<description>Cumulative Operating Time</description>
+		<category>Number</category>
```

Comment:
```suggestion
		<category>Time</category>
```

Reply:
I've updated this.  This should probably use UoM too.